keras_rs/src/layers/modeling/dot_interaction_test.py

```diff
@@ -0,0 +1,101 @@
+import keras
+from absl.testing import parameterized
+from keras import ops
+from keras.layers import deserialize
+from keras.layers import serialize
+
+from keras_rs.src import testing
+from keras_rs.src.layers.modeling.dot_interaction import DotInteraction
+
+
+class DotInteractionTest(testing.TestCase, parameterized.TestCase):
+    def setUp(self):
+        self.input = [
+            ops.array([[0.1, -4.3, 0.2, 1.1, 0.3]]),
+            ops.array([[2.0, 3.2, -1.0, 0.0, 1.0]]),
+            ops.array([[0.0, 1.0, -3.0, -2.2, -0.2]]),
+        ]
+
+        feature1 = self.input[0]
+        feature2 = self.input[1]
+        feature3 = self.input[2]
+
+        self.f11 = ops.dot(feature1[0], feature1[0])
+        self.f12 = ops.dot(feature1[0], feature2[0])
+        self.f13 = ops.dot(feature1[0], feature3[0])
+        self.f22 = ops.dot(feature2[0], feature2[0])
+        self.f23 = ops.dot(feature2[0], feature3[0])
+        self.f33 = ops.dot(feature3[0], feature3[0])
+
+    def test_layer_call_1(self):
```

Comment:
Can you rename the tests like `test_self_interaction_false_skip_gather_false` etc.